serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -95,9 +97,9 @@ An link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/j
 
 After that, the workflow application consumes the event published by the listener and sets the result field. The consumed CloudEvent contains an attribute named `kogitoprocrefid`, which holds the workflow instance ID of the workflow.
 
-The `kogitoprocrefid` attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow.
+The `kogitoprocrefid` **attribute is crucial** because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow. For more information about correlation, see the guide xref:eventing/event-correlation-with-workflows.adoc[Event correlation in {context}].
```

Comment:
```suggestion
The `kogitoprocrefid` attribute is crucial because when the correlation is not used, then this attribute is the only way for the Callback state to identify that the related CloudEvent needs to be used to resume the workflow. For more information about correlation, see xref:eventing/event-correlation-with-workflows.adoc[Event correlation in {context}].
```